app/models/role.rb

```diff
@@ -6,8 +6,7 @@ class Role < ActiveRecord::Base
   has_many :user_roles
   has_many :users, through: :user_roles
 
-  has_many :active_user_roles, -> { where("expiry_date IS NULL OR expiry_date >= ?", Time.now.in_time_zone) },
-            class_name: "UserRole"
+  has_many :active_user_roles, -> { where("expiry_date IS NULL OR expiry_date >= ?", Time.now.in_time_zone) }, class_name: "UserRole"
```

Comment:
```suggestion
  has_many :active_user_roles, -> { where("expiry_date IS NULL OR expiry_date >= ?", Time.current) }, class_name: "UserRole"
```

The difference of `.current` and `.now` is `.now` use the server's timezone, while .current use what the Rails environment is set to. If it's not set, then .current will be same as .now

Reply:
I haven't changed anything on this line